src/components/github-contributors-list/github-contributors-list.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC, useEffect, useMemo, useState } from 'react';
+import ContributorsList from './contributors-list';
+import Paragraph from '../../../src/components/paragraph/paragraph';
+import { getAllContributors, GithubContributor, GithubContributorResponse } from './github-contributors-utils';
+
+interface GithubContributorsListProps {
+  organizationName: string;
+  packageName: string;
+  text?: string;
+  showContributionAmount?: boolean;
+  excludedContributorsIds?: Set<number>;
```

Comment:
Why a set? It should still work if you have duplicates, no?

Reply:
But why would we need duplicates here?

Reply:
We don't but how come it's possible to have duplicates?

Reply:
Didn't get it)
By using Set we are just explicitly telling people that duplicates are not welcome here. Which makes sense, cause why you'd need duplicates here? It would work with just an array, but why when it can work with Set...

Reply:
I understand but why would you want to add the same contributor id to this set/array?

Reply:
Not sure, as a mistake maybe...